Formula/kubernetes-cli.rb

```diff
@@ -30,7 +30,7 @@ def install
 
     dir = "_output/local/bin/darwin/#{arch}"
     bin.install "#{dir}/kubectl"
-    bash_completion.install "contrib/completions/bash/kubectl"
+    (bash_completion/"kubectl").write `#{bin}/kubectl completion bash`
```

Comment:
`kubectl` v1.3.0 does not support the `completion` subcommand as far as I know, so I would avoid doing that before v1.4.0 becomes the new *stable*